pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -90,6 +91,11 @@ public SegmentCreationJob(Properties properties) {
       _pushLocations = null;
     }
 
+    String preprocessOutputPath = _properties.getProperty(JobConfigConstants.PREPROCESS_PATH_TO_OUTPUT);
```

Comment:
SegmentCreationJob should have no concept on the pre-process path. Also, client might want to keep the pre-process output files, so we should not always delete them

Reply:
I would say if user has specified `PREPROCESS_PATH_TO_OUTPUT`, it's ok to keep it.
If not, use a tmp dir as the preprocess output path, then remove this tmp dir when the whole workflow finished.